### PR TITLE
Start bokeh app to activate bokeh's clean session callbacks.

### LINKED ISSUE #6694: The dashboard is not cleaning Bokeh Sessions after the browser disconnects.
**What happened**:
The Dask Scheduler Dashboard CPU usage increases when the dashboard is used but it keeps being high after the user closes the dashboard.  I think that this could be related with other issues like dask/dask#8493.

Steps to reproduce: 
1. Start dask-scheduler
2. Open the browser at localhost:8787
3. Reload several times. 

You can expect the CPU usage to increase the more you reload.

**What you expected to happen**:
The CPU usage should be stable or at least decrease when the user closes the tab.

**More information:**
I've added logs to see the active Bokeh sessions
```python
#dashboard/scheduler.py
...
def connect(application, http_server, scheduler, prefix=""):
    bokeh_app = BokehApplication(
        applications, scheduler, prefix=prefix, template_variables=template_variables
    )
    application.add_application(bokeh_app)
    bokeh_app.initialize(IOLoop.current())
    bokeh_app.add_handlers(
        r".*",
        [
            (
                r"/",
                web.RedirectHandler,
                {"url": urljoin((prefix or "").strip("/") + "/", r"status")},
            )
        ],
    )

    import threading
    threading.Thread(
        target=run_stuff,
        args=(bokeh_app,),
    ).start()

from time import sleep

def run_stuff(bokeh_app):
    while True:
        sleep(1)
        sessions = bokeh_app.get_sessions("/status")
        print('Total sessions', len(sessions))
        for (c, session) in enumerate(sessions):
            print('conn:', c, session.connection_count)
```

Each time the browser reloads the number of sessions increases even though the connection count for all sessions but one is 0.

**How to fix this? (I think)**

The problem is that the cleaning callback loop is not started. To start one simply has to call `bokeh_app.start()` after the initialization.

```python
#dashboard/scheduler.py
...
def connect(application, http_server, scheduler, prefix=""):
    bokeh_app = BokehApplication(
        applications, scheduler, prefix=prefix, template_variables=template_variables
    )
    application.add_application(bokeh_app)
    bokeh_app.initialize(IOLoop.current())
    bokeh_app.add_handlers(
        r".*",
        [
            (
                r"/",
                web.RedirectHandler,
                {"url": urljoin((prefix or "").strip("/") + "/", r"status")},
            )
        ],
    )
    bokeh_app.start()
```

Also setting the option `unused_session_lifetime_milliseconds` to 500 helps cleaning the sessions faster.

**Environment**:

- Dask version: 2022.6.1
- Python version: 3.10
- Operating System: Ubuntu 22.04
- Install method (conda, pip, source): pip

### PR DESCRIPTION
**Pull request by z4m0**
Closes #6694

- [ ] Tests added / passed
- [ ] Passes `pre-commit run --all-files`

### CODE REVIEW

**Comment by GPUtester** [comment]
Can one of the admins verify this patch?

**Comment by gjoseph92** [comment]
The bokeh server is not something I'm familiar with, @jsignell are you?

**Comment by ncclementi** [comment]
@jacobtomlinson You know your way around the dashboard, any suggestions for point 2) in Ian's comment https://github.com/dask/distributed/pull/6728#issuecomment-1189571640 ?

**Comment by ian-r-rose** [comment]
> However for 2 I could not see any zombie task. Is there a way to reproduce this behaviour?

I'll see if I can come up with something -- I haven't seen it myself, I just have the generalized feeling that if we start an forever callback, as responsible users we should also stop it when done with it :)

**Review by jacobtomlinson — COMMENTED** [review]
This seems great! Thanks for the efforts here. 

I don't have any useful ideas around when to stop the server though. @ian-r-rose your snippet seems reasonable.

**Comment by ian-r-rose** [comment]
Update here: I did confirm that the various bokeh periodic callbacks keep on going forever after calling `close()` (though I didn't see any truly pathological behavior from that). So I think that making sure we clean up after ourselves is a good idea. I might amend the above suggestion to avoid an import of the optional bokeh dependency:

```python
def stop_services():
    for application in self.http_application.applications:
        if hasattr(application, "stop") and callable(application.stop):
            application.stop()
     for service in self.services.values(): 
         service.stop()
```

**Review by ian-r-rose — APPROVED** [review]
Thank you @z4m0, his is a really nice contribution! If CI is happy with this fix, I am as well. 

I see that you are hitting a known linting issue in #6743 which is unrelated to this PR.

File: distributed/node.py [hunk 1]
```diff
@@ -73,6 +73,10 @@ def start_services(self, default_listen_ip):
                 )
 
     def stop_services(self):
+        if hasattr(self, "http_application"):
```
[the remark this answers is not in the document]
Comment (z4m0) [reply distributed/node.py:76]: Yes it is but without that the test_worker_port_range was failing.

**Review by jacobtomlinson — COMMENTED** [review]
Looks like the linter isn't happy. Could you fix this up, [here are the docs on how to configure the linters to run on each commit](https://docs.dask.org/en/stable/develop.html#code-formatting). You'll likely need to run `precommit install` and `precommit run --all-files`.

**Comment by z4m0** [comment]
> Looks like the linter isn't happy. Could you fix this up, [here are the docs on how to configure the linters to run on each commit](https://docs.dask.org/en/stable/develop.html#code-formatting). You'll likely need to run `precommit install` and `precommit run --all-files`.

Ok it was fixed by #6779. Now the linter passes. The failing tests were passing in the last run though...

**Comment by jacobtomlinson** [comment]
@z4m0 thanks! That's fine, there is some [known CI instability here](https://dask.github.io/distributed/test_report.html) currently, the failures don't look related to this change.

**Pull request merged**